src/ui/screens/onboarding/Page.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { FragmentSafeAreaViewForTabBar } from '../../../ui/misc/FragmentSafeAreaView';
+import { ReactNode } from 'react-redux';
+import {
+    View,
+    TouchableWithoutFeedback,
+    StyleSheet,
+} from 'react-native';
+import {
+    DefaultTabBarHeight,
+    Colors,
+} from '../../../styles';
+import { MediumText } from '../../../ui/misc/text';
+import { ButtonProps } from '../../../components/NavigationHeader';
+
+interface Props {
+    backgroundColor: string;
+    safeAreaTopBackgroundColor?: string;
+    children: ReactNode | ReactNode[];
+    leftButton?: ButtonProps;
+    rightButton?: ButtonProps;
+}
+
+export const Page = (props: Props) => {
+    return (
+        <FragmentSafeAreaViewForTabBar
+            topBackgroundColor={props.safeAreaTopBackgroundColor}
+            bottomBackgroundColor={Colors.WHITE}
+        >
+            <View style={{
+                flex: 1,
+                backgroundColor: props.backgroundColor,
+            }}>
+                {props.children}
+            </View>
+            <View style={styles.buttonRow}>
+                {props.leftButton != null &&
+                <PageButton
+                    {...props.leftButton}
+                    color={Colors.DARK_GRAY}
+                />
+                }
+                {props.rightButton != null &&
+                <PageButton
+                    {...props.rightButton}
```

Comment:
The right button text should be aligned to right instead of left.

Reply:
Yes, we talked about it with @david-gauquelin, it'll be changed. I was not certain about it because the previous page has centered buttons.